p2p_wallet/Scenes/CreateOrRestoreWallet/Common/Pincode/Pincode/PincodeViewModel.swift

```diff
@@ -19,6 +19,7 @@ final class PincodeViewModel: BaseViewModel, ObservableObject {
     @Injected private var pincodeService: PincodeService
     @Injected private var userWalletManager: UserWalletManager
     @Injected private var notificationService: NotificationService
+    @Injected private var logoutService: LogoutService
```

Comment:
Remove this line, also, at it will trigger session scope a little bit sooner

Reply:
you're right, thanks. i didn't thought of this